listenbrainz/listen_replay/replay_user.py

```diff
@@ -0,0 +1,100 @@
+""" This script should be used to replay user listens to fix bad data.
+"""
+import abc
+import json
+import uuid
+
+from datetime import datetime
+from flask import current_app
+from listenbrainz.utils import get_measurement_name, quote, convert_to_unix_timestamp
+from listenbrainz.webserver import create_app
+from listenbrainz.webserver.influx_connection import init_influx_connection
+from listenbrainz.listenstore.influx_listenstore import DUMP_CHUNK_SIZE
+
+
+class UserReplayer(abc.ABC):
+    def __init__(self, user_name):
+        self.user_name = user_name
+        self.max_time = datetime.now()
+        self.app = create_app()
+
+
+    @abc.abstractmethod
+    def filter(self, row):
+        """ Modify row as needed by the subclass
+
+        Returns:
+            row (dict): modified row as needed
+                or None if row needs to be removed
+        """
+
```

Comment:
pass ?

Reply:
I remember reading some PEP8 / flake8 rules about not using passes for empty functions / classes with docstrings but I can't find it now. I'll add the pass.